examples/react/next-app-router/package.json

```diff
@@ -15,7 +15,8 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-instantsearch": "7.0.1",
-    "react-instantsearch-router-nextjs": "7.0.1"
+    "react-instantsearch-router-nextjs": "7.0.1",
+    "react-instantsearch-ssr-nextjs": "7.0.1"
```

Comment:
do you think we could have instead one package `react-instantsearch-nextjs` that has both?

Reply:
The router only works for `pages` while the other only works for `app` though 😅

Reply:
As long as it's experimental I'd keep it as a separate package. But I dig the idea of having a single package for nextjs specific things in the future.